src/core/Options.cpp

```diff
@@ -40,7 +40,7 @@
 
 Options::Options()
     : showWindowBorders_(true)
-    , showTouchPoints_(false)
+    , showTouchPoints_(true)
```

Comment:
the fix is in WallGraphicsScene.cpp, but I turned the option on by default because it felt useful to have them